store/sqlstore/group_store.go

```diff
@@ -734,7 +734,7 @@ func (s *SqlGroupStore) TeamMembersToAdd(since int64, teamID *string) ([]*model.
 
 	var teamMembers []*model.UserTeamIDPair
 
-	_, err = s.GetReplica().Select(&teamMembers, query, params...)
+	_, err = s.GetMaster().Select(&teamMembers, query, params...)
```

Comment:
Were these queries only used by the LDAP sync code, and therefore always pointing to master?

Reply:
Correct.